docs/index.md

```diff
@@ -0,0 +1,150 @@
+<!--
+Copyright 2024 Ekumen, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+```{image} _images/logo_with_name_dark.png
+:class: only-dark
+:alt: Shows Beluga logo.
+```
+
+```{image} _images/logo_with_name_light.png
+:class: only-light
+:alt: Shows Beluga logo.
+```
+
+---
+
+[![CI pipeline](https://github.com/Ekumen-OS/beluga/actions/workflows/ci_pipeline.yml/badge.svg?branch=main)](https://github.com/Ekumen-OS/beluga/actions/workflows/ci_pipeline.yml?query=branch:main)
+[![codecov](https://codecov.io/gh/Ekumen-OS/beluga/branch/main/graph/badge.svg?token=rK7BNC5giK)](https://codecov.io/gh/Ekumen-OS/beluga)
+[![pre-commit](https://img.shields.io/badge/pre--commit-enabled-brightgreen?logo=pre-commit)](https://github.com/pre-commit/pre-commit)
+[![License Apache-2.0](https://img.shields.io/badge/license-Apache--2.0-blue.svg)](https://github.com/Ekumen-OS/beluga/blob/main/LICENSE)
+
+# Overview
+
+```{toctree}
+:hidden:
+:maxdepth: 1
+:caption: Getting started
+
+getting-started/installation
+getting-started/quickstart
+```
+
+```{toctree}
+:hidden:
+:maxdepth: 1
+:caption: Concepts
+
+concepts/key-concepts
+concepts/design-principles
+concepts/architecture
+```
+
+```{toctree}
+:hidden:
+:maxdepth: 1
+:caption: Guides
+
+guides/using-beluga-amcl
+guides/extending-beluga
+guides/profiling-beluga
+guides/benchmarking-beluga
+```
+
+```{toctree}
+:hidden:
+:maxdepth: 1
+:caption: Tutorials
+
+tutorials/particle-filtering
+tutorials/nav2-integration
+```
+
+```{toctree}
+:hidden:
+:maxdepth: 1
+:caption: Packages
+
+beluga <packages/beluga/docs/index>
+beluga_ros <packages/beluga_ros/docs/index>
+beluga_amcl <packages/beluga_amcl/docs/index>
```

Comment:
Note that not all packages have been added to the documentation. `beluga_system_tests` and `beluga_example` seemed accessory, `beluga_benchmark` should be here but it's a bit behind our latest benchmarks. Plan is to pull it in as soon as we can migrate LAMBKIN stuff to it (WIP).